atd-vzd/views/view_crashes_inconsistent_numbers.sql

```diff
@@ -37,7 +37,7 @@ FROM atd_txdot_crashes atc
          LEFT JOIN primarypersons atpp ON atc.crash_id = atpp.crash_id
          LEFT JOIN units atu ON atc.crash_id = atu.crash_id
 WHERE 1 = 1
-  AND (austin_full_purpose = 'Y' OR (atc.city_id = 22 and atc.position IS NULL))
+  AND (atc.in_austin_full_purpose = true)
```

Comment:
i basically searched for everywhere we use the old austin_full_purpose flag in the DB and changed all those instances to using the new column. all these views will need to be recreated in staging and prod.